src/components/NewStepper/Stepper.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { ReactElement } from 'react'
+import StepperMUI from '@material-ui/core/Stepper'
+import StepMUI from '@material-ui/core/Step'
+import StepContent from '@material-ui/core/StepContent'
+import StepLabel from '@material-ui/core/StepLabel'
+import Paper from '@material-ui/core/Paper'
+import { makeStyles } from '@material-ui/core/'
+
+import Hairline from '../layout/Hairline'
```

Comment:
The following imports use an absolute path, so maybe here as well?

Reply:
done!